### ISSUE TITLE
YOLO preprocessing issues

### ISSUE THREAD

**Issue by jordi-bird** [dvazquezcvc/mcv-m5#31]
In order to solve the [YOLO preprocessing issues](https://github.com/hprop/mcv-m5/issues/2) in our implementation, we wonder if the imagenet weights employed were created applying pre-processing.
If that's the case, we would like confirm if it's the one detailed in data_loader.py as 'imagenet' or any more techniques were used.

Thanks for your time.

**Comment by lluisgomez** [comment]
@jordi-bird As far as I know the YOLO pre-trained weights are learned without any image pre-processing other than re-scaling the data (with 1./255). This is actually the case for many nets that use batch normalization.

**Comment by lfrcoder** [comment]
Hello, I saw your work on the mcv-m5 project on github and I feel very good, but there is only code on github, no data and corresponding keras development kit. Can you pass a complete data such as relevant data and keras development kit? thank you very much!
